### PR TITLE
fix: a11y parameters for text and link

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Enhancement

#### What is expected?

I added possibility to set and `id` for text to be able to anchor some titles.
I also added `aria-label` for links as apparently links should always have label: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#interactive_elements_must_be_labeled

### CODE REVIEW

File: src/components/Link/index.tsx [hunk 1]
```diff
@@ -125,6 +126,7 @@ const Link = forwardRef(
       rel,
       className,
       onClick,
+      'aria-label': ariaLabel,
```
Comment (chambo-e) [inline src/components/Link/index.tsx:129]: > for links, buttons and clickable elements, the text content of the element typically provides the label

as children is mandatory that should be enough no ?

Comment (matthprost) [reply src/components/Link/index.tsx:129]: @chambo-e Here is a good example why it is not enough sometimes:

Screenshot 2022-11-30 at 11 19 13

https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8

Comment (chambo-e) [reply src/components/Link/index.tsx:129]: Yeah but we don't do that right ? 🤔

**Review by chambo-e — APPROVED** [review]

File: src/components/Link/index.tsx (hunk 1 above)
Comment (matthprost) [reply src/components/Link/index.tsx:129]: @chambo-e we actually do with learn more links for example

**Pull request merged**